static/css/styles.css

```diff
@@ -169,6 +169,79 @@ body {
     font-weight: 600;
 }
 
+/* Holidays Section */
+.holidays-section {
+    margin-top: 1.5rem;
+    padding-top: 1rem;
+    border-top: 1px solid rgba(0, 0, 0, 0.1);
+    text-align: left;
+    width: 100%;
+}
+
+.holidays-title {
+    font-size: 0.9rem;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    color: #6b7280;
```

Comment:
Probably best to stick to one color type rather tham mixing hex with rgba. check out HSL/HSLA if you have some time. i think it's way more intuitive than rgba and of course, hex.